### PR TITLE
Upload File By Selector keyword

### PR DESCRIPTION
**Pull request by xylix**

### CODE REVIEW

File: Browser/keywords/interaction.py [hunk 1]
```diff
@@ -978,3 +979,10 @@ def keyboard_input(self, action: KeyboardInputAction, input: str, delay=0):
                 )
             )
             logger.debug(response.log)
+
+    @keyword(tags=("Setter", "PageContent"))
+    def upload_file_by_selector(self, selector: str, path: PathLike):
```
Comment (aaltat) [inline Browser/keywords/interaction.py:984]: Keyword documentation with example would be nice to have.

File: atest/test/02_Content_Keywords/files.robot [hunk 2]
```diff
@@ -8,17 +8,35 @@ Test Teardown       Run Keywords    Restore Library Timeout    AND    Wait For A
 ${CUSTOM_DL_PATH} =     ${CURDIR}/download_file
 
 *** Test Cases ***
-Upload File
+Upload upload_test_file
+    Upload Named File   test_upload_file
+
+# 75 starts to fail with firefox and chromium
```
Comment (aaltat) [inline atest/test/02_Content_Keywords/files.robot:14]: Do we need these comment lines?

File: atest/test/02_Content_Keywords/files.robot [hunk 3]
```diff
@@ -8,17 +8,35 @@ Test Teardown       Run Keywords    Restore Library Timeout    AND    Wait For A
...
+# Upload 75MB file
+#    Upload Sized File  75
+
+Upload 1MB file
+    [Tags]    no-windows-support
+    Upload Sized File  1
+    
+Upload 74MB file
+    [Tags]    no-windows-support
+    Upload Sized File  74
```
Comment (aaltat) [inline atest/test/02_Content_Keywords/files.robot:26]: Should there be test which fails on too big file?

Comment (xylix) [reply atest/test/02_Content_Keywords/files.robot:26]: Implemented at https://github.com/MarketSquare/robotframework-browser/pull/1511 directly against master

File: atest/test/02_Content_Keywords/files.robot (hunk 2 above)
Comment (xylix) [reply atest/test/02_Content_Keywords/files.robot:14]: Fixed in https://github.com/MarketSquare/robotframework-browser/pull/1511

**Pull request merged**